src/styles/main.css

```diff
@@ -0,0 +1,69 @@
+.page__body {
+  margin: 0;
+  height: 100vh;
+  background-color: #fff;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.stopwatch {
+  width: 80vmin;
+  height: 80vmin;
+  border: 1vmin dotted #000;
+  border-radius: 50%;
+  position: relative;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.stopwatch__hand--minutes {
+  background-color: #0700ff;
+  height: 20vmin;
+  width: 3vmin;
+  position: absolute;
+  top: calc(50% - 20vmin);
+  left: calc(50% - 1.5vmin);
+  z-index: 1;
+  transform-origin: 50% 100%;
+  animation: rotate 3600s steps(60) infinite;
+}
+
+.stopwatch__hand--seconds {
+  background-color: #2c8000;
+  height: 38vmin;
+  width: 1.5vmin;
+  position: absolute;
+  top: calc(50% - 38vmin);
+  left: calc(50% - 0.75vmin);
+  z-index: 2;
+  transform-origin: 50% 100%;
+  animation: rotate 60s linear infinite;
+}
+
+.stopwatch__center {
+  background-color: #f6a603;
+  height: 5vmin;
+  width: 5vmin;
+  border-radius: 50%;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  z-index: 3;
+}
+
+@keyframes rotate {
+  100% {
+    transform: rotate(360deg);
+  }
+}
+
+.stopwatch--speed-up .stopwatch__hand--seconds {
+  animation-duration: 10s;
```

Comment:
Code Quality issue: It's a good practice to avoid magic numbers in your code. Consider defining a CSS variable for this value.